cookbooks/gplates/doc/gplates.tex

```diff
@@ -0,0 +1,205 @@
+of the South American coastline. The (coarse) three-dimensional model on the other hand
+shows the same location of up- and downwellings but additionally provides a consistent solution
+that is different from the two dimensional setups. Note that the Nazca slab is subducting eastward,
+while all 2D models (even in high resolution) predict a westward subduction.
+
+Finally we would like to emphasize that these models (especially the used material model)
+are way too simplified to draw any scientific conclusion out of it. Rather it is thought
+as a proof-of-concept what is possible with the dimension independent approach of
+\aspect{} and its plugins.
+
+\begin{figure}
+  \includegraphics[width=\textwidth]{cookbooks/gplates/doc/gplates-comparison.png}
+  \hfill
+  \caption{\it Using GPlates for velocity boundary conditions: The top left figure shows
+  the results of a three-dimensional model using the present day plate velocities
+  provided by GPlates as surface boundary condition. 
+  The top right figure shows three independent computations 
+  on two-dimensional slices through Earth. The boundary conditions for each of these slices (white
+  arrows) are tangential to the slices and are projections of the
+  three-dimensional velocity vectors into the two-dimensional plane
+  occupied by the slice. While the two top models are created with the same mesh resolution
+  the bottom figure shows three independent two-dimensional models using a higher resolution. 
+  The view is centered on South America with Antarctica being near the bottom of
+  the figure (coastlines provided by NGU and the GPlates project).}
+  \label{fig:gv-1}
+\end{figure}
+
+\paragraph{Time-dependent boundary conditions.}
+\label{sec:time-dependent-gplates-velocities}
+The example presented above uses a constant velocity boundary field that
+equals the present day plate movements. For a number of purposes one may want to
+use a prescribed velocity boundary condition that changes over time, for example
+to investigate the effect of trench migration on subduction. Therefore \aspect{}'s
+GPlates plugin is able to read in multiple velocity files and linearly interpolate
+between pairs of files to the current model time. To achieve this, one needs
+to use the \texttt{\%d} wildcard in the velocity file name, which represents the current
+velocity file index (e.g. \texttt{time\_dependent.\%d.gpml}). This index is
+calculated by dividing the current model time by the user-defined time step
+between velocity files (see parameter file above). As the model time progresses
+the plugin will update the interpolation accordingly and if necessary read in
+new velocity files. In case it can not read the next velocity file, it assumes
+the last velocity file to be the constant boundary condition until the end of
+the model run. One can test this behavior with the provided data files
+\texttt{data/velocity\_boundary\_conditions/gplates/time\_dependent.\%d.gpml}
```

Comment:
```suggestion
\texttt{data/boundary\_velocity/gplates/time\_dependent.\%d.gpml}
```

Reply:
it is actually ``data/boundary-velocity/``...